### DISCUSSION TITLE
Can we access Piku's services/functions via a rest api ?

### DISCUSSION THREAD

**Discussion by Tdnshah** [piku/piku#343] [Ideas]
I would be interested in knowing if currently piku provides any rest end points to add/delete project, the whole aim it create a webui for a simple hosting like  platform based on Piku be created?

**Comment by chr15m** [comment]
Piku's deployment method is `git push` so you are going to need the command line anyway. If you want a web interface for managing apps it would be fairly trivial to implement as a Piku app itself.
